examples/project-images.html

```diff
@@ -0,0 +1,19 @@
+<script type="module">
+    import {HanskenClient} from './src/hansken.js';
+   
+    const params = {};
+    window.location.search.substr(1).split('&').forEach((param) => {
+        const parts = param.split('=');
+        params[parts[0]] = decodeURIComponent(parts[1]);
+    });
+
+    const projectId = params.projectId;
+    if (projectId) {
+        const hansken = new HanskenClient('/gatekeeper');
+        hansken.project(projectId).images().then((images) => {
+            images.forEach((image) => {
+                document.write(`<li>${image.description}`);
+            });
+        });
+    }
+</script>
```

Comment:
No newline at end of file here, but some other files do. To help avoid unnecessary changes this should be consistent for all files (maybe add a `.editorconfig`?).

Reply:
Added new lines